packages/commcare/test/index.js

```diff
@@ -679,11 +678,131 @@ describe('HTTP wrappers', () => {
       },
     };
 
-    const { data, response } = await 
-      request('GET', '/a/asri/api/v0.5/case')
-    (state);
+    const { data, response } = await request(
+      'GET',
+      '/a/asri/api/v0.5/case'
+    )(state);
 
     expect(data).to.haveOwnProperty('case_id');
     expect(response.statusCode).to.equal(200);
   });
-});
\ No newline at end of file
+});
+
+describe('Bulk', () => {
+  it('should submit JSON as an xls file', async () => {
+    let formdata;
+
+    testServer
+      .intercept({
+        path: `/a/${domain}/importer/excel/bulk_upload_api/`,
+        method: 'POST',
+      })
+      .reply(200, req => {
+        // save out the form data that was upladed
+        formdata = req.body;
+
+        // simulate a return from commcare
+        return { code: 200, message: 'success' };
+      });
+
+    const state = {
+      configuration: {
```

Comment:
Since this is use in couple of places you can tidy this up my putting 
configuration in it's own const at the top
```js
const configuration = {
        hostUrl,
        domain,
        appId: app,
        username: 'user',
        password: 'password',
    };
 ```
 Then you can setup state
 ```js
 const state = { configuration, data: {//some initial data} }
 ```